sdk/keyvault/azure-security-keyvault-keys/vcpkg/vcpkg.json

```diff
@@ -18,7 +18,7 @@
     {
       "name": "azure-core-cpp",
       "default-features": false,
-      "version>=": "1.9.0"
+      "version>=": "1.12.0-beta.1"
```

Comment:
I can understand why we do Identity - next week we will release a post-GA Beta Identity which will depend on post-GA Beta Core, not sure about keyvault keys, I do not think we are releasing it this month? By the time we release, it is possible that a non-beta 1.12.0 of Core is in vcpkg official.

Reply:
I think it's worth keeping the dependency graph "honest" regardless of when we ship the packages, as the files exist in the repo. Whenever we ship keyvault keys, we need to be clear, it needs to depend on a version of core >= 1.12.0-beta.1, since there is an actual dependency here.

We should certainly update the dependency to whatever latest version that is required (favoring the GA'd version if possible).